apps/core-angular-universal/src/index.html

```diff
@@ -6,6 +6,7 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="stylesheet" href="https://unpkg.com/@cds/core@5.4.2/global.min.css" />
```

Comment:
Why are we pulling our global stylesheet in from `unpkg.com`?

Reply:
Yeah its because of the build issue below. The current workaround doesnt work for universal apps that I can see. I can add a comment here to refer to the same issue